kpops/components/base_components/pipeline_component.py

```diff
@@ -160,6 +160,24 @@ def inflate(self) -> list[PipelineComponent]:
         """
         return [self]
 
+    def template(self, api_version: str, ca_file: str, cert_file: str) -> None:
+        """Runs `helm template`
+
+        From HELM: Render chart templates locally and display the output.
+        Any values that would normally be looked up or retrieved in-cluster will
+        be faked locally. Additionally, none of the server-side testing of chart
+        validity (e.g. whether an API is supported) is done.
+
+        Args:
+            api_version (str): Kubernetes api version used for
+                Capabilities.APIVersions, `--api_versions` in helm
+            ca_file string (str): verify certificates of HTTPS-enabled servers
+                using this CA bundle, `--ca-file` in helm
+            cert_file (str): identify HTTPS client using this SSL certificate
+                file, `--cert-file` in helm
+        """
```

Comment:
it's great that you added a docstring 🙂 
For consistency, please use the same format (reST) that we are using in other places. Is this the Google style?

Reply:
Yess, this was Google style. I should've indeed checked what was already there.

I looked for the reST guide, but the information I found was not very clear on the details and was somewhat scattered. Please lmk if you notice any inconsistencies.
(That's one thing I really like about the Google guide, it's very comprehensive and easy to navigate.)

Reply:
looks good to me :) we're also not 100% consistent about docstrings yet, if you have ideas how to improve it we can discuss it in the team